doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -24,9 +24,11 @@
 <a name="2.1"></a>
 ## 2.1 Introduction
 
-**must**: Requirements that are marked as _must_ are considered mandatory and must exist in the reference architecture and reflected in any implementation targeting this reference architecture. The same applies to _must not_.
+**must**: Requirements that are marked as _must_ are considered mandatory and must exist in the Reference Architecture and reflected in any Reference Implementation or Vendor Implementation targeting this Reference Architecture. The same applies to _must not_.
```

Comment:
nit: targeting => compliant / conformant to

Reply:
good spot, have changed it now. Please resolve and approve if you're happy.